tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh

```diff
@@ -140,7 +140,7 @@ for vol in "$POOL/vol" "$POOL/$FS/vol" ; do
 	rand_set_prop $vol checksum "on" "off" "fletcher2" "fletcher4" "sha256"
 	rand_set_prop $vol compression "on" "off" "lzjb" "gzip" \
 		"gzip-1" "gzip-2" "gzip-3" "gzip-4" "gzip-5" "gzip-6"   \
-		"gzip-7" "gzip-8" "gzip-9"
+		"gzip-7" "gzip-8" "gzip-9" "zstd"
```

Comment:
"off gzip lz4 lzjb zstd" should also be enough here. 
all those gzip levels are useless, gzip-6 is the same as gezip, on is the same as lz4 atm. if we are making a change here, we could just as well remove unusefull or (at least) the duplicate algorithms